manifests/t/Thales/SafeNetAuthenticationClient/10.9.3283.0/Thales.SafeNetAuthenticationClient.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Thales.SafeNetAuthenticationClient
+PackageVersion: 10.9.3283.0
+InstallerLocale: en-US
+InstallerType: zip
+NestedInstallerType: msi
+ReleaseDate: 2024-08-28
+Installers:
+- Architecture: x86
+  InstallerUrl: https://www.digicert.com/StaticFiles/Windows_SAC_10.9_GA.zip
+  InstallerSha256: 3B963D24C0B145775B8ECD950D8568DC44EBA7A872792C009D0287DCD7AAC034
+  NestedInstallerFiles:
+  - RelativeFilePath: Msi/x32/SafeNetAuthenticationClient-x32-10.9.msi
+- Architecture: x64
+  InstallerUrl: https://www.digicert.com/StaticFiles/Windows_SAC_10.9_GA.zip
+  InstallerSha256: 3B963D24C0B145775B8ECD950D8568DC44EBA7A872792C009D0287DCD7AAC034
+  NestedInstallerFiles:
+  - RelativeFilePath: Msi/x64/SafeNetAuthenticationClient-x64-10.9.msi
+- Architecture: arm64
+  InstallerUrl: https://www.digicert.com/StaticFiles/Windows_SAC_10.9_GA.zip
+  InstallerSha256: 3B963D24C0B145775B8ECD950D8568DC44EBA7A872792C009D0287DCD7AAC034
+  NestedInstallerFiles:
+  - RelativeFilePath: Msi/arm64/SafeNetAuthenticationClient-arm64-10.9.msi
```

I thought I had clicked "Start a review" to add this to the above review, but must have forgotten.
```suggestion
  - RelativeFilePath: SAC 10.9 GA/Msi/x32/SafeNetAuthenticationClient-x32-10.9.msi
- Architecture: x64
  InstallerUrl: https://www.digicert.com/StaticFiles/Windows_SAC_10.9_GA.zip
  InstallerSha256: 3B963D24C0B145775B8ECD950D8568DC44EBA7A872792C009D0287DCD7AAC034
  NestedInstallerFiles:
  - RelativeFilePath: SAC 10.9 GA/Msi/x64/SafeNetAuthenticationClient-x64-10.9.msi
- Architecture: arm64
  InstallerUrl: https://www.digicert.com/StaticFiles/Windows_SAC_10.9_GA.zip
  InstallerSha256: 3B963D24C0B145775B8ECD950D8568DC44EBA7A872792C009D0287DCD7AAC034
  NestedInstallerFiles:
  - RelativeFilePath: SAC 10.9 GA/Msi/arm64/SafeNetAuthenticationClient-arm64-10.9.msi
```